tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const_canvas3d';
+
+context('Canvas 3D functionality. Add cuboid.', () => {
+    const caseId = '64';
+
+    const screenshotsPath = 'cypress/screenshots/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js';
```

Comment:
It's better to not use file extension in folder name.

Reply:
In this form, cypress creates directories for placing screenshots.